### PR TITLE
Exclude sandbox previews from visual test screenshots

### PR DESCRIPTION
**Pull request by diegohaz**
## Summary
- Filter out sandbox entries from the `/previews` API endpoint so visual tests only screenshot example previews, not sandboxes
- Sandbox preview pages remain accessible; only the endpoint consumed by the visual test is affected

## Test plan
- [ ] Verify visual test no longer takes screenshots of sandbox previews (e.g., `counter`, `5518`)
- [ ] Verify sandbox preview pages still load correctly at their URLs

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: site/src/pages/previews.ts [hunk 1]
```diff
@@ -3,10 +3,12 @@ import { getCollection } from "astro:content";
...
 export const GET: APIRoute = async () => {
   const entries = await getCollection("previews");
-  const paths = entries.flatMap((entry) =>
-    entry.data.frameworks.map(
-      (framework) => `/${framework}/previews/${entry.id}`,
-    ),
-  );
+  const paths = entries
+    .filter((entry) => !entry.filePath?.includes("/sandbox/"))
+    .flatMap((entry) =>
```
[the remark this answers is not in the document]
Comment (diegohaz) [reply site/src/pages/previews.ts:8]: Fixed in ae916da27 — switched to a regex `/(^|\/)sandbox\//.test(...)` to also handle the edge case where `filePath` starts with `sandbox/` without a leading slash.

File: site/src/pages/previews.ts [hunk 2]
```diff
@@ -3,10 +3,12 @@ import { getCollection } from "astro:content";
 
 export const GET: APIRoute = async () => {
   const entries = await getCollection("previews");
-  const paths = entries.flatMap((entry) =>
-    entry.data.frameworks.map(
-      (framework) => `/${framework}/previews/${entry.id}`,
-    ),
-  );
+  const paths = entries
+    .filter((entry) => !entry.filePath?.includes("/sandbox/"))
```
[the remark this answers is not in the document]
Comment (diegohaz) [reply site/src/pages/previews.ts]: Fixed in ae916da27 — switched to a regex that matches the `sandbox` path segment. That said, the Windows concern doesn't apply here: Astro's glob loader computes `filePath` via `posixRelative()`, which always produces forward-slash paths regardless of platform.
